packages/core/src/lib/normalize.unit.test.ts

```diff
@@ -0,0 +1,112 @@
+import { join } from 'node:path';
+import { describe, expect } from 'vitest';
+import { AuditOutput, Issue } from '@code-pushup/models';
+import { normalizeAuditOutputs, normalizeIssue } from './normalize';
+
+vi.mock('@code-pushup/utils', async () => {
+  const actual = await vi.importActual('@code-pushup/utils');
+
+  return {
+    ...actual,
+    gitRoot: vi.fn().mockResolvedValue('/User/code-pushup'),
+  };
+});
+
```

Comment:
The mock doesn't do anything. There is no `gitRoot` export, the two exported functions being used by `normalizeAuditOuputs` are called `getGitRoot` and `formatGitPath`.

```suggestion
```

Reply:
I will remove the mock at all as if I would want to check that the value returned by the mock is passed to `normalizeIssue` i cant do it in the same file.

Reply:
Now the mock has a use